Doc/whatsnew/3.12.rst

```diff
@@ -253,6 +253,13 @@ asyncio
 * :func:`asyncio.wait` now accepts generators yielding tasks.
   (Contributed by Kumar Aditya in :gh:`78530`.)
 
+csv
+---
+
+* Add :data:`~csv.QUOTE_NOTNULL` and :data:`~csv.QUOTE_STRINGS` flags to
+  provide finer control treatment of ``None`` and empty strings by
```

Comment:
control or treatment? 😉

Reply:
Thanks. "control", not "treatment."

I have never used this edit-thru-github feature before and can't figure out how to edit the file again. Also, I see the need for adding "csv." to the data references. What is the tilde for?

Reply:
tilde will make sphinx display only the last part of the dotted name

Reply:
quick edit on github PR:
- go to files tab
- hover over a line
- click `+` button next to line number to comment
- click `±` toolbar button to add a so-called suggestion in your comment
- edit the code in the suggestion, check the preview to see the diff
- publish comment
- commit the change, or leave for the reviewer